playbooks/bastion/README.md

```diff
@@ -0,0 +1,38 @@
+# Bastion Host / Control Host playbook
+
+This playbook uses a variety of roles in this repo to setup a bastion host, also some times called a control host. The inventory can be used (per instructions below) to control which software and services get installed on the bastion host. 
+
+
+## Prerequisites
+A running instance (VM or cloud image) such as Fedora, CentOS or Red Hat Enterprise Linux. The instance needs to be subscribed (if applicable) and configured with access to the necessary repos (in most cases, the exsisting repos / configuration is sufficient).
+
+## Gotcha's 
+1. If running in a cloud environment, for example OpenStack, make sure to have the correct ports open in the security groups (e.g.: 5901 for VNC, 22 for SSH, etc.)
+2. When enabling VNC, and you already have a shared home directory, make sure the proper changes are made to the VNC configuration (typically in `~/.vnc` ) to allow for the service to run correctly.
+
+## Example run
+How to run the playbook may depend on the options selected. However, below is an example execution whereas the password for IPA/IdM integration (with `ipa_install` set to "yes" in the inventory) is passed in rather than statically set in the inventory. Modify the inventory to your liking in `playbooks/bastion/inventory`, then at the top level of the repository, execute the following command:
+
+```
+> ansible-playbook -i playbooks/bastion/inventory playbooks/bastion/install.yml -e 'ipa_password=<ipa/IdM password>'
+```
+
+**Note:** If your password contains any special characters, e.g.: a '!', it's important to use the single quotes for the passed in value as it otherwise may be interpereted by the shell. 
+
+## Inventory Options
+
+| variable | info |
+|:--------:|:----:|
+|main_user|The username this bastion is primerly being enabled for| 
+|ipa_install|Set to "yes" if you'd like to integrate with a backend IPA/IdM service|
+|ipa_domain|If `ipa_install` is set to "yes", set this domain to match what your environment uses (obtain from sys admin if not known)|
```

Comment:
Well, that doesn't make sense. This domain is a pre-defined domain already in use by the environment you are installing into , so it's not what "you'd like to use in your IPA server". I've added a few notes around familiarity with IdM/IPA and "consult your sys admin" for details when/if not known.